src/job.ts

```diff
@@ -22,6 +22,8 @@ export class CronJob<OC extends CronOnCompleteCommand | null = null, C = null> {
 		: undefined;
 	waitForCompletion = false;
 	errorHandler?: CronJobParams<OC, C>['errorHandler'];
+	threshold = 250; // default threshold in ms
+	name?: string; // optional job name for identification
```

Comment:
not a critical issue but it does bother me a bit that these two variables are not always in the same order 🙃

Reply:
I feel you :laughing: